crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -80,6 +80,9 @@ enum Command {
         /// Only a developer testing flag, as it might be needed for testing.
         #[clap(long, parse(try_from_str = parse_human_readable_size))]
         max_plot_size: Option<u64>,
+        /// Actually write pieces to disk while benchmarking (might be more accurate)
+        #[clap(long)]
+        plot: bool,
```

Comment:
This is not a great name. I can imagine something like `write_to_disk` that is an enum with variants that allow writing nothing, everything, just plot, just auxiliary databases, etc. Can start with this:
```
enum WriteToDisk {
    Everything,
    Nothing,
}